### PR TITLE
Remove GET support from ML server

### PR DESCRIPTION
**Pull request by milesgranger**
Problem :car: 
---
We have a `GET` endpoint for the ML server which has gone unused for some time and has shown no use for continued support.

Solution :truck: 
---
Remove it.

### CODE REVIEW

**Review by epa095 — COMMENTED** [review]
Feels good to delete right? I think quite some code can be deleted from the CLI as well (all that takes inn influx credentials for the server)

**Comment by milesgranger** [comment]
@epa095 guess we can ignore the patch coverage? :sweat_smile:

**Comment by flikka** [comment]
We are sure that Lennart is not planning on using this in his client, right?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: gordo_components/server/views/anomaly.py [hunk 1]
```diff
@@ -49,10 +42,6 @@
     "tags": fields.Nested(_tags),
     "total_abnormality": fields.Float,
 }
-API_MODEL_OUTPUT_GET = api.model(
-    "Prediction - Output from GET",
-    {"output": fields.List(fields.Nested(_single_prediction_record))},
-)
 
 
 class AnomalyView(BaseModelView):
```
Comment (epa095) [inline gordo_components/server/views/anomaly.py:47]: Docstring is wrong now (get)

File: gordo_components/server/views/anomaly.py [hunk 2]
```diff
@@ -93,6 +82,8 @@ class AnomalyView(BaseModelView):
      'time-seconds': '0.1937'}
     """
 
+    methods = ["POST"]
```
Comment (epa095) [inline gordo_components/server/views/anomaly.py:83]: I cant see where this variable is used. If it is a constant maybe it should BE CAPS!?!?!

File: gordo_components/client/client.py [hunk 3]
```diff
@@ -290,18 +290,10 @@ def predict(
               1st element is the dataframe of the predictions; complete with a DateTime index.
               2nd element is a list of error messages (if any) for running the predictions
         """
-
```
Comment (epa095) [inline gordo_components/client/client.py:293]: Docstring for this method must be updated

File: gordo_components/server/views/anomaly.py (hunk 1 above)
Comment (epa095) [reply gordo_components/server/views/anomaly.py:47]: Also holds for other methods in the file

**Review by epa095 — COMMENTED** [review]
Mostly LGTM, but docstrings must be looked over

**Review by epa095 — APPROVED** [review]
I am going to prematurely approve this(!) since it is only docstring changes, and I have faith in you

File: gordo_components/server/views/anomaly.py (hunk 2 above)
Comment (milesgranger) [reply gordo_components/server/views/anomaly.py:83]: Flask uses it internally in their [Views](https://flask.palletsprojects.com/en/1.1.x/api/#flask.views.View.methods) to auto return [405](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405)s

**Pull request merged**